src/drivers/distance_sensor/ll40ls/CMakeLists.txt

```diff
@@ -40,4 +40,6 @@ px4_add_module(
 		LidarLite.cpp
 		LidarLiteI2C.cpp
 		LidarLitePWM.cpp
+	DEPENDS
+		drivers_rangefinder
 	)
```

Comment:
Hi @dagar, can you add `STACK_MAIN 1200` to the CMakeLists.txt? (Or allocate more if you prefer.)